components/script/dom/window.rs

```diff
@@ -1356,7 +1402,9 @@ impl Window {
     }
 
     pub fn get_url(&self) -> Url {
-        (*self.Document().url()).clone()
+        let doc = self.Document();
+        let val = (*doc.url()).clone();
+        val
```

Comment:
@nox I pulled his branch and tried that too, but to no avail. I hear that it's because of dropck.